drivers/include/sx126x.h

```diff
@@ -92,6 +105,47 @@ uint32_t sx126x_get_channel(const sx126x_t *dev);
  */
 void sx126x_set_channel(sx126x_t *dev, uint32_t freq);
 
+/**
+ * @brief   Converts symbol value to milliseconds.
+ *
+ * @param[in] dev                      Device descriptor of the driver
+ * @param[in] symbols                  Symbols
+ *
+ * @return Time for symbols in milliseconds
+ */
+static inline int sx126x_symbol_to_msec(sx126x_t *dev, uint16_t symbols)
+{
+    assert(dev && (dev->mod_params.bw <= SX126X_LORA_BW_500) && \
+           (dev->mod_params.bw >= SX126X_LORA_BW_125));
+
+    /* Refer section 4.1.1.7  Time on air in SX1276/77/78/79 datasheet */
+    return (symbols * (1 << (dev->mod_params.sf + 7 - dev->mod_params.bw)) / 1000);
+}
+
+/**
+ * @brief   Check if onboard SUBGHZ Radio is being used
+ *
+ * @param[in] dev                      Device descriptor of the driver
+ *
+ * @return True, if onboard SUBGHZ Radio is being used
+ */
+static inline bool IS_SUBGHZ(sx126x_t *dev)
```

Comment:
I thought this was already renamed to `sx126x_is_stm32wl`, as discussed offline

Reply:
ping

Reply:
This is a single commit [a639bdd](https://github.com/RIOT-OS/RIOT/pull/16579/commits/a639bdd01f283fd7fc1e3360a73f0b1328b4e916). Shall remove it in the next rebase :)